src/cdk-experimental/menu/context-menu.ts

```diff
@@ -108,6 +110,10 @@ export class CdkContextMenuTrigger implements OnDestroy {
     return this._menuPanel;
   }
   set menuPanel(panel: CdkMenuPanel) {
+    // If the provided panel already has a stack, that means it already has a trigger configured
+    if (panel._menuStack && isDevMode()) {
```

Comment:
Note that the `isDevMode` calls will become lint errors with https://github.com/angular/components/pull/20146. I suspect that your changes will land first though.

Reply:
ya, I saw that and wasn't sure how to address it. I think I'll issue a follow up PR after it lands - I'll add a TODO for now